src/현정빈/Myclass.java

```diff
@@ -0,0 +1,148 @@
+package 현정빈;
+
+import jun.ManagementSystem;
+import jun.Member;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class Myclass implements ManagementSystem {
+    HashMap<Integer,Member> map = new HashMap<>(); //컬렉션 중 Map을 선택하여 HashMap 사용
+    // 키는 integer 타입, 값은 Member 타입인 키-값쌍 구조의 map 생성
+
+    /**
+     * 저장소에 멤버를 추가하는 메서드,
+     * 이미 존재하는 id라면 추가에 실패해야 함,
+     * 추가가 성공하면 true를 반환
+     */
+    @Override
+    public boolean addMember(String name, int age, int id) {
+        Member newMember = new Member(name, age, id);
+        if(!map.containsKey(id)) {//지정된 id를 포함하고 있는지 여부를 반환, 반환값이 false라면
+            map.put(id, newMember); //map에 지정된 키(id)와 값(객체 newMember)을 저장
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
```

Comment:
현재 코드는 아이디가 중복되더라도 일단 멤버 객체를 생성되는 코드로 확인됩니다! 이렇게 되면 아이디가 같은 멤버 객체가 생성될 수도 있겠죠? 또한 `else` 예약어를 안쓰는 연습도 해보면 좋을 것 같아요~ 

```suggestion
    @Override
    public boolean addMember(String name, int age, int id) {
        if (member.containsKey(id)) {
            return false;
        }
        member.put(id, new Member(name, age, id));
        
        return true;
    }
```

이렇게 `else`문 없이 `early return` 구조를 사용하면 더 좋은 코드가 될 것 같아요! 아래의 `else` 예약어를 사용한 메서드들도 한 번 리펙토링 해보시면 좋을 것 같습니다!
아래 블로그를 참고해 보세요~
* https://limdingdong.tistory.com/8